src/test/java/uk/ac/cam/cl/dtg/isaac/api/QuizFacadeTest.java

```diff
@@ -290,13 +290,13 @@ public void getQuizAssignmentAttempt() {
                 prepare(quizAssignmentManager, m -> expect(m.getGroupForAssignment(studentAssignment)).andReturn(studentGroup)),
                 prepare(associationManager, m -> expect(m.hasPermission(currentUser(), student)).andReturn(true)),
                 prepare(quizAttemptManager, m -> expect(m.getByQuizAssignmentAndUser(studentAssignment, student)).andReturn(completedAttempt)),
-                prepare(quizQuestionManager, m -> expect(m.augmentQuestionsForUser(studentQuiz, completedAttempt, true)).andReturn(augmentedQuiz)),
-                prepare(assignmentService, m -> {
-                    m.augmentAssignerSummaries(Collections.singletonList(studentAssignment));
-                    expectLastCall();
-                }),
+                prepare(quizQuestionManager, m -> expect(m.augmentFeedbackFor(completedAttempt, studentQuiz, QuizFeedbackMode.DETAILED_FEEDBACK)).andReturn(augmentedQuiz)),
+//                prepare(assignmentService, m -> {
+//                    m.augmentAssignerSummaries(Collections.singletonList(studentAssignment));
+//                    expectLastCall();
+//                }),
```

Comment:
I'm assuming this will eventually be removed due to the other changes you made, or is there a particularly hairy reason why you had to comment this one out?

Reply:
I meant to ask you to have a look if you could - I couldn't quite figure out how to make this bit of the test work, or if it's actually no longer needed with the new question augmenting I added

Reply:
I'll have a look, thanks.